📜 Add a `LICENCE` to ensure composability and re-use virality

JFWooten4 opened the pull request:
@tehchives recently added the awesome DRS percent holding leaderboard image code from @jergy2 in e67a074beb8811261698f3e475417dfd6925f5f9.

I was just glancing over this earlier today in finding pages to share in #23. This was one of my favorite pages, but I didn't experiment with it because I figured it'd have a custom social preview of the actual image.

Anyway, since we're adding content here to decentralize the ongoing public info development, I suppose it'd be prudent to include a license earlier rather than later—so that we don't have to query everyone that's done work in the future like in WhyDRS/Taking-Stock#17.

As long as that sounds good via an `OK` from Chives and Jersan, then this PR should get that set up for everyone!

JFWooten4 commented:
Also, James was kind enough to let me know in 808552d5146d687a2a34c2f71d702454d88589a8 that we aren't using some of the default imports from the Wix bot, so I'll add another commit removing these deprecated boilerplate functions.

tehchives commented:
What you're describing sounds good to me, but I might be missing something basic about Git navigation here. Looking at the PR, the licensing language was already present excepting a [^def] (and what is that?) while most of the changes are removing an example. Curious to know more about the changes you are performing in this PR and their implications if you do not mind! Thank you John for your forward thinking about how we can license these growths mindfully for tomorrow!

JFWooten4 commented:
Sure, so I removed firstsly some default scipts that we aren't using ebcuase those are not created by sany of the ocmmuntiy members. Accodingnyl, they would preusmably be the proprietary code of Wix or their Velo development team - as thye came from the intial importt through the site inteface. This could be problamaic because they were added without any kind of licence, so I think then safest be t is to just remove the m given we are not using the very minor aspects iwth then.

As for the [^def], that was just reformatting the `README` explainign the permissions JSON. I removed a redundant example which sdusplicates their existing public documentation and migreated a repetative sentence into the footnote. That's how footnotes work in markdown:

```md
This is my text.[^fn]

[^fn]: Pretty short!
```

You can try this out in a comment reply if you'd lke, adn see it when you press `Preview`. I would prefer to delete the whole file, but you can't add comments in the relevant p[eremission file due to its structure, so I figured we could leave the baic info about what it's doing for now. 

So yes, you are right in that a lot of the changes are removign examples, largely because these examples lie in a licencing grey-zone where we don't know who wrote them, and building on them later could be toubleson-ish. So my impelmentaiton opted for maximum safety when introducing our own Afferro by ensuring that only the material code we wrote still standsin the repo. That and this onerrequisired very bried permissions security config that ties into the cental site for now.

Upon merging, it'd also gbe great ot get this OK'd by Jersan since I understand that they wrote some of the code which is being ocmmited, which is why I tegged them,. I'd say the same for anything else that was developmedd by the handful of communtiy memebers iwth access to the Wix login.

Happy to get it on the table! I went through some SYndicate rpos and added a few licences last night, too. just the kind of thing that is 1000% easier to do at the start to set the ground rules rather than retroactively working around everythign later, notfying anyone who helped, and generally offering a lot of ambigruity for potential open-source contributors who happened to like the repo. All thsi came to me when my `free-,markets` project got its first star, so I just wanted to make sure this was up to scale given the awesome addition of the custom leaderboard code. Presumabily, we can do a whole lot more with this repo than just hese (unusued) script if awesome communtiy emmebrs like Jersoan keep sharing their custom implementations here!

tehchives commented:
Makes a lot more sense! Thank you. Very reasonable to wait for @jergy2 input as well.

jergy2 commented:
If I am understanding the situation correctly then I have no issues whatsoever with proceeding.  Please proceed!  While I appreciate being included I do not feel that my approval was necessary for proceeding in this case.  Let me know if there are any issues and I will assist
